abixen-platform-business-intelligence-service/src/main/java/com/abixen/platform/service/businessintelligence/multivisualisation/application/service/impl/ChartConfigurationManagementServiceImpl.java

```diff
@@ -119,15 +102,20 @@ public ChartConfiguration build(final ChartConfigurationForm chartConfigurationF
                 .build();
     }
 
+    @Override
+    public void delete(final Long moduleId) {
```

Comment:
Call it deleteChartConfiguration - bacause in the application layer we need full names